style.css

```diff
@@ -43,24 +43,205 @@ nav {
   position: absolute;
   right: 1rem;
   display: none;
+  top: 1.5rem;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 30px;
+  height: 21px;
+}
+
+.close {
+  display: none;
 }
 
 @media (max-width: 500px) {
   .menu-button {
     display: flex;
   }
 
+  .close-button {
+    width: 30px;
+    height: 30px;
+    left: 5px;
+    top: 5px;
+  }
+
+  .close {
+    display: flex;
+    height: 50px;
+    width: 100%;
+    border-radius: 8px;
+    justify-content: flex-end;
+    padding: 25px;
+  }
+
   .navbar-links {
     display: none;
+    width: 105%;
+    background: rgba(96, 112, 255, 0.5);
+    height: 100vh;
+    margin-left: -20px;
   }
 
   .navbar {
+    flex-direction: column;
+    align-items: flex-start;
     padding-left: 20px;
   }
+
+  .navbar-links ul {
+    width: 100%;
+    flex-direction: column;
+  }
+
+  .navbar-links li {
+    padding-top: 50px;
+    top: 70px;
+  }
+
+  .navbar-links li a {
+    padding: 0.5rem 1rem;
+    color: #fff;
+    font-weight: 600;
+    font-size: 32px;
+    line-height: 44px;
+  }
+
+  .navbar-links.active {
+    display: flex;
+  }
```

Comment:
- As the last code [reviewer suggested](https://github.com/schoolsofafrika/Responsive-Portfolio/pull/5#discussion_r1030268544) kindly make sure that the background of the popup menu has a blurred effect. :eyes: :eyes: 

Your design
![image](https://user-images.githubusercontent.com/98363075/203549215-cc6944e3-0dda-44f4-bd5c-2c27379ce793.png)

Figma design
![image](https://user-images.githubusercontent.com/98363075/203549285-4759193a-5785-45c1-993b-9c42643d13dd.png)

- As the last [code reviewer](https://github.com/schoolsofafrika/Responsive-Portfolio/pull/5#discussion_r1030268544) suggested you should not have a vertical scroll on the menu popup. Kindly fix this. :pray: :pray: 

![image](https://user-images.githubusercontent.com/98363075/203549692-48c11412-3076-467d-9806-911f4fc1f8f7.png)